src/main/resources/db/migration/V4_4__increase_arsaker_size.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE AKTIVITETSKRAV_VURDERING
```

Comment:
Må øke størrelsen på denne da kolonnen ble for liten når man krysset av for alle årsakene (vi lagrer arsak-enum-verdiene som en kommaseparert string).